pkg/network/tracer/tracer.go

```diff
@@ -887,7 +887,7 @@ func (t *Tracer) GetNetworkID(context context.Context) (string, error) {
 }
 
 const connProtoTTL = 3 * time.Minute
-const connProtoCleaningInterval = 5 * time.Minute
+const connProtoCleaningInterval = 65 * time.Second // slight jitter to avoid all maps cleaning at the same time
```

Comment:
This change can be contributing to inaccuracies in USM
Please run that on oddish-c for a day, so we can evaluate if it impacts our accuracy

Reply:
that is a fair concern, I'll do that today. How do you feel about the current TTL value of 3 minutes?

Reply:
I think that's problematic as well, but changing those two values all together would cause more problem for us in the short term.

We need to revisit the design and implementation of the map, to see how we can handle better the connections, as we can (and we have) long living connections (mainly around http2) that can suddenly disappear (due to the cleaner). But on the other hand, the map is leaking or growing too fast, so we need to evaluate maybe normalizing the key (and cutting the number of actual entries by 50%, or increasing our actual capacity by 100%), and having multiple values instead just the protocol.

Reply:
Here are the results of [the test in oddish](https://ddstaging.datadoghq.com/dashboard/kfn-zy2-t98/agent-system-probe-overview-v2?fromUser=true&refresh_mode=paused&tpl_var_kube_cluster_name%5B0%5D=oddish-c&from_ts=1730955062640&to_ts=1731083874000&live=false). What do you think about the results? It's hard for me to interpret them as I don't have context on USM telemetry.

I agree that 3 minutes seems quite small for connections as many can be longer lived. How would we further normalize the key?

Reply:
I reviewed the code again, and I was wrong in my analysis
The cleaner removed inactive connection (a.k.a the last time we changed `val.Updated` is older than `connProtoTTL`)
In that case, I don't see a risk to change the running interval to 1m instead of 5m (besides cpu possible CPU increase)